src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -428,13 +428,19 @@ PerfAnalyzer::WriteReport()
   bool should_output_metrics{
       params_->should_collect_metrics && params_->verbose_csv};
 
+  // (TMA-1526) Detect if the model is LLM and report LLM metrics based on that
```

Comment:
put FIXME or TODO so it is obvious this is a comment to be followed up on.
Also, the new story to hook it all up is 1557

Reply:
Fixed the comment